sites/public/src/pages/listings.tsx

```diff
@@ -58,7 +58,8 @@ export function getServerSideProps() {
       listingsEndpoint: getListingServiceUrl(),
       googleMapsApiKey: runtimeConfig.getGoogleMapsApiKey(),
       // show Bloom counties by default
-      initialSearch: "counties:Alameda,San Mateo,Santa Clara",
+      initialSearch:
+        "counties:Alameda,Contra Costa,Marin,San Francisco,San Mateo,Santa Clara,Solano,Sonoma",
```

Comment:
Should we add Napa as well?

Reply:
@ifranch great point! I didn't even notice that Napa wasn't on there (rushing too quickly). It doesn't exist in the Filter modal right now.  @isaac-looker will need to ensure it exists in the new modal build out PR.

Reply:
It's funny how we've demoed that a few times and no one has noticed. 😱

Reply:
Does it sound ok by you, @ifranch, if we do that in a subsequent PR?